app/serializers/api/admin/order_serializer.rb

```diff
@@ -8,6 +11,48 @@ def full_name
     object.billing_address.nil? ? "" : ( object.billing_address.full_name || "" )
   end
 
+  def distributor_name
+    object.distributor.andand.name
+  end
+
+  def show_path
+    return '' unless object.id
```

Comment:
what about `object.id || ''`?

Reply:
The object id isn't being returned by that method...

Reply:
:poop: right. I shut up then